aspnetcore/fundamentals/logging/index.md

```diff
@@ -2,584 +2,351 @@
 
-The second `AddFilter` specifies the Debug provider by using its type name. The first `AddFilter` applies to all providers because it doesn't specify a provider type.
+### Default log level
 
-### How filtering rules are applied
+If the default log level is not set, the default log level value is `Information`.
 
-The configuration data and the `AddFilter` code shown in the preceding examples create the rules shown in the following table. The first six come from the configuration example and the last two come from the code example.
+For example, consider the following web app:
 
-| Number | Provider      | Categories that begin with ...          | Minimum log level |
-| :----: | ------------- | --------------------------------------- | ----------------- |
-| 1      | Debug         | All categories                          | Information       |
-| 2      | Console       | Microsoft.AspNetCore.Mvc.Razor.Internal | Warning           |
-| 3      | Console       | Microsoft.AspNetCore.Mvc.Razor.Razor    | Debug             |
-| 4      | Console       | Microsoft.AspNetCore.Mvc.Razor          | Error             |
-| 5      | Console       | All categories                          | Information       |
-| 6      | All providers | All categories                          | Debug             |
-| 7      | All providers | System                                  | Debug             |
-| 8      | Debug         | Microsoft                               | Trace             |
+* Created with the ASP.NET web app templates.
+* *appsettings.json* and *appsettings.Development.json* deleted or renamed.
 
-When an `ILogger` object is created, the `ILoggerFactory` object selects a single rule per provider to apply to that logger. All messages written by an `ILogger` instance are filtered based on the selected rules. The most specific rule possible for each provider and category pair is selected from the available rules.
+With the preceding code, navigating to the privacy or home page produces many `Trace`, `Debug`, and `Information`  messages with `Microsoft` in the category name.
 
-The following algorithm is used for each provider when an `ILogger` is created for a given category:
+The following code sets the default log level when the default log level is not set in configuration:
 
-* Select all rules that match the provider or its alias. If no match is found, select all rules with an empty provider.
-* From the result of the preceding step, select rules with longest matching category prefix. If no match is found, select all rules that don't specify a category.
-* If multiple rules are selected, take the **last** one.
-* If no rules are selected, use `MinimumLevel`.
+[!code-csharp[](index/samples/3.x/MyMain/Program.cs?name=snippet_MinLevel&highlight=9)]
 
-With the preceding list of rules, suppose you create an `ILogger` object for category "Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine":
+<!-- review required: I say this a couple times -->
```

Comment:
This seems sensible to me, but it's more guidance than technical, so I know that doesn't count for much. :) But yeah, this makes it much easier to dial up logging levels when a live system is under investigation.